.github/workflows/build-deb-package.yml

```diff
@@ -0,0 +1,69 @@
+name: Build Debian package
+on:
+  push:
+    branches:
+      - main
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  build-deb-package:
+    strategy:
+      matrix:
+        target:
+          - { python: "3.6.15", ubuntu: 20.04 }
+          - { python: "3.10.12", ubuntu: 22.04 }
```

Comment:
Why do we need a separate pipeline for that? We can reuse existing workflow and build a .deb package every commit like we do with .whl files

Reply:
Fixed